app/graphql/mutations/library_entry/create.rb

```diff
@@ -0,0 +1,29 @@
+class Mutations::LibraryEntry::Create < Mutations::Base
+  argument :input,
+    Inputs::LibraryEntry::Create,
+    required: true,
+    description: 'Create a Library Entry',
+    as: :library_entry
+
+  field :library_entry, Types::LibraryEntry, null: true
+
+  def load_library_entry(value)
+    LibraryEntry.new(value.to_h)
+  end
+
+  def authorized?(library_entry:)
```

Comment:
Where is this called? if it’s automatically called, how is library_entry: passed in?  In Ruby 3, they will be separating hash parameters from named parameters, so if you use a named parameter here you need to make sure that it’s a named parameter (not a hash!) wherever it’s called

Reply:
So the argument `input` is aliased using `as: :library_entry`. The `#load_library_entry` will grab the record if we override it, and pass that through as `library_entry: <loaded item>`. 
From there graphql has hooks for a `#ready?` (this one is before `#load_library_entry` funny enough) and `#authorized?` (after `#load_library_entry`). For `#authorized?` it injects the argument directly into as `library_entry:` due to us aliasing it. (if we didn't alias, it would be `input:` then). After that it will send the record directly the `#resolve` exactly the same way it did to authorized.

I did leave this comment in one of the mutations, unsure where it should actually be because all of them use this:
```
  # NOTE: https://github.com/rmosolgo/graphql-ruby/issues/1733
  # NOTE: https://www.rubydoc.info/github/rmosolgo/graphql-ruby/GraphQL/Schema/Resolver#argument-class_method
  # the argument loads: ... is not required, it is called every time.
```

Sidenote: I'll need to ready about what Ruby 3 is doing with named parameters vs hash parameters.

Reply:
This shouldn't cause an issue when we go to Ruby 3